### PR TITLE
chores(pull_request_template): change instructions and link templates

### PR DESCRIPTION
**Pull request by OleksiyRudenko**
An attempt to improve clarity of instructions in `.github/pull_request_template.md`

Please feel free suggesting further improvements.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
-_Complete the form below and remove this line._
+_Have your app published on GitHub pages. Make sure your PR title contains the task name. Change task name and links below as appropriate. Remove this paragraph._
```
Comment (babayK0) [inline .github/pull_request_template.md:1]:
```suggestion
_Have your app published on [GitHub pages](https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/getting-started-with-github-pages). Make sure your PR title contains the task name. Change task name and links below as appropriate. Remove this paragraph._
```

Maybe add link to github pages guide/FAQ ?

**Review by babayK0 — APPROVED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (OleksiyRudenko) [reply .github/pull_request_template.md:1]: Agree, but perhaps a different, more lightweight and straightfoward guide.
I was looking at this particular... It addresses some irrelevant matters like individual website (username.github.io) built with with `.md` files, Jekyll themes etc.

Just cannot recall a short guide on publishing small web apps I came across once.

Comment (OleksiyRudenko) [reply .github/pull_request_template.md:1]: BTW, looks like we have that link somewhere else. This is only explanation why some of the students publish their apps on their "main", personal github pages )

Comment (OleksiyRudenko) [reply .github/pull_request_template.md:1]: @babayK0 will introduicng a brief and focused guide like #225 help? What do you think?

Comment (babayK0) [reply .github/pull_request_template.md:1]: > @babayK0 will introduicng a brief and focused guide like #225 help? What do you think?

Nice! It is more understandable and simple than guide which i provided)

Comment (OleksiyRudenko) [reply .github/pull_request_template.md:1]:
```suggestion
_Have your app published on [GitHub pages](https://github.com/kottans/frontend-2021-homeworks/blob/main/publish-your-app.md). Make sure your PR title contains the task name. Change task name and links below as appropriate. Remove this paragraph._
```

**Pull request merged**